sos/report/plugins/logs.py

```diff
@@ -118,7 +118,7 @@ class IndependentLogs(LogsBase, IndependentPlugin):
     profiles = ('system', 'hardware', 'storage')
 
 
-class CosLogs(LogsBase, CosPlugin):
+class CosLogs(IndependentLogs, CosPlugin):
```

Comment:
Does this mean that you don't want to be collecting all the logs defined in `LogsBase` as above?

Reply:
This will make it so that `CosLogs` has the `IndependentPlugin` tagging class assigned to it, which will enable it for _all_ distros, which is precisely why we made the change to split out `LogsBase`.

The change to `valid_subclasses` is correct (and can be made "more" correct by removing line 36), but this change here is not going to get us where we need to be.